content/docs/rfcs/32/README.md

```diff
@@ -101,7 +101,11 @@ The signal hash can be generated by hashing the raw signal (or content) using th
 
 #### External nullifier
 
-The external nullifier can be generated by hashing a raw string (i.e UNIX timestamp) value using `keccak256`.
+The external nullifier SHOULD be computed as the Poseidon hash of the current epoch (i.e the current UNIX timestamp divided by the epoch length) and RLN identifier.
```

Comment:
```suggestion
The external nullifier MUST be computed as the Poseidon hash of the current epoch (i.e the current UNIX timestamp divided by the epoch length) and RLN identifier.
```
Imo mentioning MUST is required here since messages would be invalidated if the peer doesn't compute the external_nullifier in this way.

Reply:
Thanks!